docs/installation/linux.md

```diff
@@ -634,8 +634,11 @@ Optionally you can create a symbolic link to be able to start PowerShell without
 
 ```sh
 # Start PowerShell with sudo to create a symbolic link
```

Comment:
I think you should make clear that the two lines below need to run from bash (otherwise the escaping of `$` won't work correctly below).  Just change this comment to:

```sh
# Start PowerShell from bash with sudo to create a symbolic link
```

Reply:
Updated.